src/tests/performance/ping.rs

```diff
@@ -0,0 +1,210 @@
+    //  ├╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┤
+    //  │   300 ┆  1000 ┆        0 ┆       81 ┆           6 ┆       11 ┆       12 ┆       13 ┆       14 ┆       52 ┆    13.45 ┆ 22312.15 │
+    //  ├╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┤
+    //  │   500 ┆  1000 ┆        0 ┆      107 ┆           7 ┆       18 ┆       20 ┆       21 ┆       22 ┆       61 ┆    21.79 ┆ 22943.68 │
+    //  ├╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┤
+    //  │   750 ┆  1000 ┆        0 ┆      119 ┆           8 ┆       27 ┆       30 ┆       32 ┆       34 ┆       63 ┆    32.39 ┆ 23156.53 │
+    //  ├╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┤
+    //  │   800 ┆  1000 ┆        0 ┆      137 ┆           8 ┆       29 ┆       32 ┆       33 ┆       36 ┆       68 ┆    34.53 ┆ 23167.10 │
+    //  ╰───────┴───────┴──────────┴──────────┴─────────────┴──────────┴──────────┴──────────┴──────────┴──────────┴──────────┴──────────╯
+    //
+    //  zebra
+    //  ╭───────┬───────┬──────────┬──────────┬─────────────┬──────────┬──────────┬──────────┬──────────┬──────────┬──────────┬─────────╮
+    //  │ Peers ┆ Pings ┆ Min (ms) ┆ Max (ms) ┆ stddev (ms) ┆ 10% (ms) ┆ 50% (ms) ┆ 75% (ms) ┆ 90% (ms) ┆ 99% (ms) ┆ Time (s) ┆  Ping/s │
+    //  ╞═══════╪═══════╪══════════╪══════════╪═════════════╪══════════╪══════════╪══════════╪══════════╪══════════╪══════════╪═════════╡
+    //  │     1 ┆  1000 ┆        0 ┆        4 ┆           1 ┆        1 ┆        1 ┆        1 ┆        1 ┆        2 ┆     1.79 ┆  559.10 │
+    //  ├╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌┤
+    //  │    10 ┆  1000 ┆        0 ┆      128 ┆           6 ┆        3 ┆        4 ┆        5 ┆        6 ┆       47 ┆     5.77 ┆ 1733.04 │
+    //  ├╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌┤
+    //  │    20 ┆  1000 ┆        0 ┆       79 ┆           7 ┆       19 ┆       20 ┆       20 ┆       21 ┆       44 ┆    21.44 ┆  932.71 │
+    //  ├╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌┤
+    //  │    30 ┆  1000 ┆        0 ┆      178 ┆          12 ┆       49 ┆       51 ┆       53 ┆       55 ┆      104 ┆    54.24 ┆  553.09 │
+    //  ╰───────┴───────┴──────────┴──────────┴─────────────┴──────────┴──────────┴──────────┴──────────┴──────────┴──────────┴─────────╯
+
+    // number of pings to send per peer
+    const PINGS: usize = 1000;
```

Comment:
Did you try a lower value with a higher peer count?

Reply:
Results with `PINGS=100` are comparable to `PINGS=1_000`, but have more variation. Likely due to lower overall runtime and time to average out.